remoto/tests/test_log.py

```diff
@@ -36,6 +36,30 @@ def test_strip_new_lines(self):
         message = conn.logger.error.call_args[0][0]
         assert message == 'an error message'
 
+    def test_strip_new_lines_extra_1(self):
```

Comment:
ideally test method names would describe a little bit of what it is testing. If you could update these to reflect that it would be much better.

For example, instead of `test_strip_new_lines_extra_1` it could be `test_strip_new_line`.

And instead of `test_strip_new_lines_extra_2` it could be `test_strip_new_line_and_carriage_return`